src/common/layers/Layer.js

```diff
@@ -15,7 +15,7 @@ import { v4 as uuid } from 'uuid';
  * @classproperty {boolean} isQueryable - Define if the layer can be queried. Read-only.
  * @classproperty {boolean} isReactSpatialLayer - Custom property for duck typing since `instanceof` is not working when the instance was created on different bundles. Read-only.
  * @classproperty {Layer[]} children - List of children.
- * @classproperty {string} copyright - Copyright.
+ * @classproperty {String[]} copyrights - Array of copyrights.
```

Comment:
Can't we use {String|String[]}  setting is a string is more easy to a user.

Reply:
```suggestion
 * @classproperty {Array<string>} copyrights - Array of copyrights.
```